composer.json

```diff
@@ -43,7 +43,7 @@
         "ext-libxml": "*",
         "composer/xdebug-handler": "^1.3",
         "justinrainbow/json-schema": "^5.2",
-        "nikic/php-parser": "^4.1",
+        "nikic/php-parser": "^4.2",
```

Comment:
What do you think about making it explicit that we require at least `nikic/php-parser:^4.2.1`?

Reply:
done